doc/weights.xml

```diff
@@ -0,0 +1,70 @@
+#############################################################################
+##
+#W  weights.xml
+#Y  Copyright (C) 2014-19                               Raiyan Chowdhury
+##
+##  Licensing information can be found in the README file of this package.
+##
+#############################################################################
+##
+
+
+<#GAPDoc Label="EdgeWeights">
+    <ManSection>
+    <Attr Name="EdgeWeights" Arg="digraph"/>
+    <Oper Name="EdgeWeightsMutableCopy" Arg="digraph"/>
+    <Returns>A list of lists of integers, floats or rationals.</Returns>
+    <Description>
+     <C>EdgeWeights</C> returns the list of lists of edge weights of
+    each edge of the digraph <A>digraph</A>.
+
+    More specifically, a value <C>j</C> appears in <C>weights[i]</C> each time
+    there exists the edge <C>[i, j]</C> in <A>digraph</A>. <P/>
```

Comment:
This doesn't sound right – sounds more like the description of `OutNeighbours`.

You'd want to say something like:
```
More specifically, <C>weights[i][j]</C> is the weight given to the <C>j</C>th edge from vertex <C>i</C>, according to the ordering of edges given by <C>OutNeighbours(digraph)[i]</C>.
```